src/mds/MDLog.cc

```diff
@@ -1487,7 +1491,6 @@ void MDLog::_replay_thread()
       if (sb->is_major_segment_boundary()) {
         major_segments.insert(event_seq);
         logger->set(l_mdl_segmjr, major_segments.size());
-        events_since_last_major_segment = 0;
```

Comment:
need to update `minor_segments_since_last_major_segment` here too, no?

Reply:
good catch, yes!